dxr/templates/nunjucks/promoted.html

```diff
@@ -0,0 +1,27 @@
+{% macro promoted_list(query, promoted, promoted_count, www_root, tree) -%}
+  {% for result in promoted %}
+    <div class="result-head">
+      <div class="left-column">
+        <div class="icon-container {{ result.iconClass }}"></div>
+      </div>
+      <div>
+        {{ result.pathLine | safe }}
+      </div>
+    </div>
+  {% endfor %}
+{%- endmacro %}
+
+{%- macro promoted_header(promoted_query, promoted_count, promoted_count_formatted, top_of_tree, tree) -%}
+  <p class="promoted-see-more">
+    {% if promoted_count == 1 %}
+      Found 1 path result:
+    {% elif promoted_count <= 5 %}
+      Showing all of <a href="{{ promoted_query }}">{{ promoted_count_formatted }} path results</a>:
+    {% else %}
+      Showing first few of <a href="{{ promoted_query }}">{{ promoted_count_formatted }} path results</a>:
```

Comment:
I can't remember what we were thinking before, but we might as well do the conventional thing here and say...

```
Path results:
* blah
* blah
* blah

___37 more...___
```
Can you think of a reason not to?